packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -857,6 +862,67 @@ export class GarbageCollector implements IGarbageCollector {
 		return this.summaryStateTracker.refreshLatestSummary(result);
 	}
 
+	/**
+	 * Process a GC message.
+	 * @param message - The GC message from the container runtime.
+	 * @param local - Whether it was send by this client.
+	 */
+	public processMessage(message: ISequencedDocumentMessage, local: boolean) {
+		const gcMessageContents = message.contents as IGarbageCollectionMessage;
+		switch (gcMessageContents.type) {
+			case GarbageCollectionMessageType.Sweep: {
+				// Delete the nodes whose ids are present in the contents.
+				const sweepMessageContents = gcMessageContents.contents;
+				this.deleteSweepReadyNodes(sweepMessageContents.deletedNodeIds);
+				break;
+			}
+			default: {
+				break;
```

Comment:
Let's put a comment here highlighting that this means future GC Message Types will be acceptable to old clients, but ignored.

Reply:
Did just that. Take a look.

Reply:
Ok, gotcha. At first I was picturing adding a 2nd compatDetails inside the GC message. But that does seem redundant; the compatDetails applies to the whole op, regardless of which part of the op may be unrecognized.